src/java/org/apache/cassandra/index/sai/StorageAttachedIndex.java

```diff
@@ -90,17 +99,35 @@
 import org.apache.cassandra.schema.ColumnMetadata;
 import org.apache.cassandra.schema.IndexMetadata;
 import org.apache.cassandra.schema.TableMetadata;
+import org.apache.cassandra.service.ClientWarn;
 import org.apache.cassandra.service.StorageService;
 import org.apache.cassandra.utils.Pair;
 import org.apache.cassandra.utils.concurrent.OpOrder;
 
+import static org.apache.cassandra.index.sai.disk.v1.IndexWriterConfig.MAX_TOP_K;
+
 public class StorageAttachedIndex implements Index
 {
     public static final String NAME = "sai";
+    
+    public static final String VECTOR_USAGE_WARNING = "SAI ANN indexes on vector columns are experimental and are not recommended for production use.\n" +
+                                                      "They don't yet support SELECT queries with:\n" +
+                                                      " * Consistency level higher than ONE/LOCAL_ONE.\n" +
+                                                      " * Paging.\n" +
+                                                      " * No LIMIT clauses.\n" +
+                                                      " * PER PARTITION LIMIT clauses.\n" +
+                                                      " * GROUP BY clauses.\n" +
+                                                      " * Aggregation functions.\n" +
+                                                      " * Filters on columns without a SAI index.\n" +
+                                                      "Most of these limitations are planned to be removed in future versions.";
```

Comment:
After recent discussions on the viability of supporting CL>ONE/LOCAL_ONE and paging, maybe we should just remove this line for now.

Reply:
Agreed, we don't need to say this.